handlers/SuiChainHandler.ts

```
@@ -14,52 +45,502 @@ export class SuiChainHandler extends BaseChainHandler<SuiChainConfig> {
+    
+    if (this.config.useEndpoint || !this.suiClient) {
+      logger.info(
+        `Sui L2 Listeners skipped for ${this.config.chainName} (using Endpoint: ${this.config.useEndpoint} or client not initialized: ${!this.suiClient}).`,
+      );
+      return;
+    }
+
+    try {
+      // SUI doesn't support WebSocket subscriptions, so we use polling
+      const POLLING_INTERVAL_MS = 5000; // Poll every 5 seconds
+      
```

ESLint

[error] Delete `······`

(prettier/prettier)